CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Changelog
 
-## Unreleased
-### Backend
-- Add municipalities and werkingsgebieden in the area of Brussel [OP-3669]
-- Updated mu-search configuration with source [OP-3670]
+## 1.37.0 (2025-10-24)
+- Added cleanup job to clean duplicated data of municipalities and provinces [OP-3676]
 
-### Deploy notes
+#### CLI
```

Comment:
in the deploy instructions you also need to add restart on the migrations service